lib/carin_for_blue_button_test_kit/docs/carin_for_blue_button_v2.0.0_client_suite_description.md

```diff
@@ -0,0 +1,106 @@
+The CARIN for Blue Button v2.0.0 Client test suite verifies the conformance of client systems to the STU2 HL7® FHIR®
+[CARIN for Blue Button® Implementation Guide](http://hl7.org/fhir/us/carin-bb/STU2). These tests are intented to validate 
+CARIN IG for Blue Button Requestors, or applications that initiate a data access request to retrieve patient data.
+This can be thought of as the client in a client-server interaction.
+
+Any actor acting a FHIR Client in this IG SHALL:
+  - Be able to process and retain all profile data elements that have a minimum cardinality >= 1 and/or flagged
+  as Must Support as defined by that profiles StructureDefinition
+  - Conform to the US Core Client Capability Statement expectations for that profiles type
```

Comment:
This feels out of place to me here, I think because it is a bit too detailed. Curious where it came from. I suspect that if we want to keep it, it would go into the Test Methodology section.

I think the second bullet is covered by what I proposed below for an update to that Test Methodology section. The first one is what I think we wrap currently into a single attestation at the end, right? I'd probably add to that list I proposed something like

>Processed and retained all profile instances and data elements marked as required or must support via tester attestation.